quickstarts/microsoft.compute/vm-simple-linux/azuredeploy.parameters.json

```diff
@@ -9,7 +9,7 @@
       "value": "GEN-PASSWORD"
     },
     "dnsLabelPrefix": {
-      "value": "GEN-UNIQUE"
+      "value": "hello-cloud"
```

Comment:
I don't think we want to change this behavior from `GEN-UNIQUE` which will generate a unique value for this `dnsLabelPrefix` value. Otherwise, this template won't work for others once the dns label is taken.